eng/scripts/Language-Settings.ps1

```diff
@@ -23,17 +25,38 @@ function Get-AllPackageInfoFromRepo($serviceDirectory)
   {
     if (!$projectOutput) { continue }
 
-    $pkgPath, $serviceDirectory, $pkgName, $pkgVersion, $sdkType, $isNewSdk = $projectOutput.Split(' ',[System.StringSplitOptions]::RemoveEmptyEntries).Trim("'")
-
+    $pkgPath, $serviceDirectory, $pkgName, $pkgVersion, $sdkType, $isNewSdk, $dllFolder = $projectOutput.Split(' ',[System.StringSplitOptions]::RemoveEmptyEntries).Trim("'")
     if(!(Test-Path $pkgPath)) {
       Write-Host "Parsed package path `$pkgPath` does not exist so skipping the package line '$projectOutput'."
       continue
     }
-    $pkgProp = [PackageProps]::new($pkgName, $pkgVersion, $pkgPath, $serviceDirectory)
+    $pkgProp = [PackageProps]::new($pkgName, $pkgVersion, $pkgPath, $serviceDirectory, $namespaces)
     $pkgProp.SdkType = $sdkType
     $pkgProp.IsNewSdk = ($isNewSdk -eq 'true')
     $pkgProp.ArtifactName = $pkgName
 
+    # Add a step to extract namespaces
+    if (!(Test-Path $dllFolder)) {
+      $dllFolder = (Split-Path $dllFolder -Parent) + "/Release"
```

Comment:
Why are you adding Release here? Debug or Release should already be included in the outputpath.

Reply:
If we want to avoid Debug or Release and just check any we should use `BaseOutputPath` instead of `OutputPath`, see https://github.com/Azure/azure-sdk-for-net/blob/main/Directory.Build.props#L28.

Reply:
Done